docs/manual/installation/install-contao.en.md

```diff
@@ -81,12 +89,40 @@ Example: `example.com`points to the directory `/www/example/web`
 Pro Contao installation therefore requires its own (sub)domain.
 {{% /notice %}}
 
+
 ### Update database tables
+After installation, you can update the database using the Contao [install tool](/en/installation/contao-installtool/). 
 
-After the installation is before the database update, call the [Contao install tool](../contao-installtool/) or use (from Contao 4.9)
+Since Contao 4.9 you can use the following command on the command line:
 
 ```bash
 php vendor/bin/contao-console contao:migrate
-```
+``` 
+
+{{% notice tip %}}
+You can also create a database beforehand on the command line:<br>
+`php vendor/bin/contao-console doctrine:database:create`
+{{% /notice %}}
+
+{{% notice info %}}
+Contao needs to know the corresponding connection data for your database. This information can either be retrieved via 
+an existing "config/paramters.yml" (currently installed using the [Contao-Install tool](/en/installation/contao-installtool/) 
+) or via a "[.env](https://docs.contao.org/dev/getting-started/starting-development/#application-configuration)" file 
+in the root directory of your installation.<br><br> 
+For details on the necessary environment variables ([DATABASE_URL](https://docs.contao.org/dev/reference/config/#database-url) 
+and [APP_SECRET](https://docs.contao.org/dev/reference/config/#app-secret)) in a ".env" file you can find 
+[here](https://docs.contao.org/dev/getting-started/starting-development/#application-configuration).
+{{% /notice %}}
+
+
+### Creating Contao backend users
+
+Using the [Contao-Installtool](/en/installation/contao-installtool/), you can create your backend user. Since Contao 
+4.10.x you can use the following command on the command line:
```

Comment:
```suggestion
### Creating Contao back end users

Using the [Contao-Installtool](/en/installation/contao-installtool/), you can create your back end user. Since Contao 
**4.10** you can use the following command on the command line:
```